components/dashboards-web-component/src/utils/apis/AuthenticationAPI.jsx

```diff
@@ -0,0 +1,91 @@
+/*
+ *  Copyright (c) 2017, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ *  WSO2 Inc. licenses this file to you under the Apache License,
+ *  Version 2.0 (the "License"); you may not use this file except
+ *  in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing,
+ *  software distributed under the License is distributed on an
+ *  "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ *  KIND, either express or implied.  See the License for the
+ *  specific language governing permissions and limitations
+ *  under the License.
+ *
+ */
+
+import Axios from 'axios';
+import Qs from 'qs';
+
+// TODO: Get following configurations from the deployment.yaml at the backend.
+/**
+ * Constants.
+ * @type {{}}
+ */
+const constants = {
+    basePath: window.location.origin,
+    grantTypePassword: 'password',
+    appContext: window.contextPath.substr(1),
+    authorizationHeader: 'Basic YWRtaW46YWRtaW4=',
```

Comment:
Any reason for hard coding ?

Reply:
Hard-coded that because of the login/logout API calls required the auth header (as mentioned in PR description). But now we don't need that. Will remove that.